.github/workflows/scripts/changelog-release-helper.test.mjs

```diff
@@ -54,6 +54,11 @@ describe('Changelog release helper', () => {
           badVersion: '3.0.2',
           type: 'patch',
           goodVersion: '3.0.1'
+        },
+        {
```

Comment:
Can we pass the `increments` array to an `it.each` test?

At the mo, if item 1 fails, you'll not get any results for the rest of the items because the whole test fails.